src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -204,6 +217,15 @@ export default class AircraftController {
      */
     addItem = (item) => this.aircraft.list.push(item);
 
+    /**
+     * Get the instance of AircraftCommander that was created by this Controller
+     *
+     * @returns {AircraftCommander}
+     */
+    getAircraftCommander() {
```

Comment:
I'd encourage moving this to a getter:

```suggestion
get aircraftCommander() {
    return ths._aircraftCommander;
}
```

if you do, make sure it's grouped with other `getter`s in this file (if any).

Reply:
moved to the proper location (right beneath constructor).